src/components/views/rooms/SlateMessageComposer.js

```diff
@@ -0,0 +1,489 @@
+/*
```

Comment:
Hmm, it's unfortunate this isn't detected as a file move... Any guess why it wasn't? If it's moved and edited, maybe the move can happen in a separate commit?

Reply:
Because I copied it over from `develop` after having heavily modified the original on my branch ... That would have been better, I only thought how to keep the slate editor functioning at the end :man_facepalming: As this code will be removed anyway, maybe it is fine?

Reply:
Hmm, depends how you feel about surgery with `git rebase -i` and such... I do find it nice to have things recorded as moves where possible... I am happy to spend a few minutes helping attempt it, as I've done this sort of thing quite often. (But yes, you are correct that the file should be short lived if everything goes according to plan... Sometimes it doesn't though... 😅)